src/manifest.rs

```diff
@@ -91,7 +132,12 @@ impl Target {
         let name: String = j["name"].take_string().expect("name is missing");
         let src_path: PathBuf =
             Path::new(&j["src_path"].take_string().expect("src_path is missing")).to_path_buf();
-        let edition: Option<String> = j["edition"].take_string();
+        let edition: Edition = match j["edition"].as_str() {
```

Comment:
Since we are calling this block twice, perhaps it can be a method on `Edition` (or `FromStr` instance) now.

Reply:
I agree